packages/fluentui/docs/src/prototypes/FormValidation/Formik.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { Form } from '@fluentui/react-northstar';
+import { Formik } from 'formik';
+
+const FormValidateFormik = () => (
+  <Formik
```

Comment:
Just wondering why did you prefer this API over hooks? https://jaredpalmer.com/formik/docs/api/useFormik

Reply:
2 reasons, internally our @types/react would have to be updated to support formik latest version and also just to have it as comparison agains other prototype that is already hooks.